### PR TITLE
MNT Fix behat test

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripe/.github/issues/349

Fixes https://github.com/silverstripe/silverstripe-elemental/actions/runs/12342587285/job/34442697226#step:12:258

    Then I click "MyPage" in the "#Form_EditForm" element                                # SilverStripe\BehatExtension\Context\BasicContext::iClickInTheElement()
      "MyPage" not found

Broke because of the new 'Archived' badge showing in the gridfield

### CODE REVIEW

File: tests/Behat/features/restore-archived-page.feature [hunk 1]
```diff
@@ -26,7 +26,7 @@ Feature: Restore to draft
     And I press the "Unpublish and archive" button, confirming the dialog
     And I go to "/admin/archive"
     Then I should see "MyPage" in the "#Form_EditForm" element
-    Then I click "MyPage" in the "#Form_EditForm" element
+    Then I click on the ".col-Title" element
```
Comment (GuySartorelli) [inline tests/Behat/features/restore-archived-page.feature]:
```suggestion
    Then I click "MyPage" directly in the "#Form_EditForm .col-Title" element
```
This matches the `restore-archived-element.feature` file which this one was mostly copied from - see https://github.com/silverstripe/silverstripe-elemental/blob/e1f612732a7f6bb223d779405df4a2b615382891/tests/Behat/features/restore-archived-element.feature#L30-L32

It has the added bonus that it specifically looks for the named page.

**Review by GuySartorelli — CHANGES_REQUESTED** [review]

File: tests/Behat/features/restore-archived-page.feature [hunk 2]
```diff
@@ -26,7 +26,7 @@ Feature: Restore to draft
     And I press the "Unpublish and archive" button, confirming the dialog
     And I go to "/admin/archive"
     Then I should see "MyPage" in the "#Form_EditForm" element
```
Comment (GuySartorelli) [inline tests/Behat/features/restore-archived-page.feature]:
```suggestion
    Then I should see "MyPage" in the "#Form_EditForm .col-Title" element
```
Optional additional change to better match `restore-archived-element.feature` and to ensure the `"MyPage"` we should see is the same as the `"MyPage"` we click.

**Review by GuySartorelli — CHANGES_REQUESTED** [review]

File: tests/Behat/features/restore-archived-page.feature (hunk 2 above)
Comment (emteknetnz) [reply tests/Behat/features/restore-archived-page.feature]: Updated

**Review by GuySartorelli — APPROVED** [review]
LGTM

**Pull request merged**